kong/clustering/wrpc_control_plane.lua

```diff
@@ -153,7 +156,7 @@ function _M:export_deflated_reconfigure_payload()
 end
 
 function _M:push_config_one_client(client)
-  if not self.config_call_rpc or not self.config_call_args then
+  if isempty(self.clients) or not self.config_call_rpc or not self.config_call_args then
```

Comment:
a comment on why do we need `isempty()` scenairo here will be helpful for future us

Reply:
Added, thanks!